opm/input/eclipse/EclipseState/Grid/FieldProps.hpp

```diff
@@ -405,31 +460,66 @@ class FieldProps {
             }
         }
 
+        /// Access underlying property data elements
+        ///
+        /// Returns \c nullptr if property data is not well formed.
         const std::vector<T>* ptr() const
         {
             return (this->data_ptr != nullptr)
                 ? &this->data_ptr->data
                 : nullptr;
         }
 
+        /// Access underlying property data elements
+        ///
+        /// Throws an exception as outlined in \c GetStatus if property data
+        /// is not well formed.
         const std::vector<T>& data() const
         {
             this->verify_status();
             return this->data_ptr->data;
         }
 
+        /// Read-only access to contained FieldData object
+        ///
+        /// Throws an exception as outlined in \c GetStatus if property data
+        /// is not well formed.
         const Fieldprops::FieldData<T>& field_data() const
         {
             this->verify_status();
             return *this->data_ptr;
         }
 
+        /// Property validity predicate.
+        ///
+        /// Returns true if propery exists and has well defined data
```

Comment:
suggestion: s/well/properly or the likes. the term 'well' is a bit reserved in the context.